### PR TITLE
register Certificate Login App Feature

### PR DESCRIPTION
**Pull request by ivanbogdanov**

### CODE REVIEW

**Comment by ivanbogdanov** [comment]
@bhariharan Fixed. Let me know

File: libs/SalesforceSDK/src/com/salesforce/androidsdk/config/RuntimeConfig.java [hunk 1]
```diff
@@ -78,6 +80,10 @@ private RuntimeConfig(Context ctx) {
 				SalesforceSDKManager.getInstance().registerUsedAppFeature(FEATURE_MDM);
 			}
 
+			if(getBoolean(RuntimeConfig.ConfigKey.RequireCertAuth)){
```
Comment (bhariharan) [inline libs/SalesforceSDK/src/com/salesforce/androidsdk/config/RuntimeConfig.java]: Move this inside the `if(isManaged && configurations!=null && !configurations.isEmpty())` block.

**Comment by ivanbogdanov** [comment]
@wmathurin

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**